frontend-project/src/pages/cases/new/index.tsx

```diff
@@ -1,9 +1,26 @@
 import { PageHeaderWrapper } from '@ant-design/pro-layout';
 import { Button, Col, Card, Form, Input, Row, Select } from 'antd';
 import { connect } from 'dva';
-import React, { useEffect } from 'react';
+import React, { useEffect, FunctionComponent } from 'react';
 import { formatMessage, FormattedMessage } from 'umi-plugin-react/locale';
 
+interface UserType {
+  username: string;
+  email: string;
+  firstName: string;
+  lastName: string;
+  id: number;
+}
+
+interface TagType {
```

Comment:
Czemu TagType, a nie po prostu np. Tag?
Skoro nie eksportujesz tego interfejsu, to możesz dać mu "prostszą" nazwę.
A jeśli w przyszłości będzie eksportowany, to nazwa TagType jest zła, bo 
a) stosuje coś w stylu notacji węgierskiej, co nie ma aktualnie większego sensu
b) jest mylna, bo to interfejs, a nie typ
Jak zmienisz nazwę to ten plik będzie ok

Reply:
+ docelowo interface będzie pewnie importowany ze store